makefiles/release.mk

```diff
@@ -213,10 +213,18 @@ bump-fluvio: install-fluvio-package
 bump-fluvio-stable: CHANNEL_TAG=stable
 bump-fluvio-stable: VERSION=$(REPO_VERSION)
 bump-fluvio-stable: bump-fluvio
+	export PKGSET_NAME=$(VERSION)
+	export FLUVIO_VERSION=$(VERSION)
+	export FLUVIO_CLOUD_VERSION="0.2.15"
+	./actions/publish-pkgset.sh
 
 bump-fluvio-latest: CHANNEL_TAG=latest
 bump-fluvio-latest: VERSION=$(subst -$(GIT_COMMIT_SHA),+$(GIT_COMMIT_SHA),$(DEV_VERSION_TAG))
 bump-fluvio-latest: bump-fluvio
+	export PKGSET_NAME=$(VERSION)
+	export FLUVIO_VERSION=$(VERSION)
+	export FLUVIO_CLOUD_VERSION="0.2.15"
```

Comment:
Same with previous, but if these values might differ, we should use different vars for the makefile.

Unless this is just a short-term fix, I would suggest having a default values set on the service-side. Otherwise we have deep coupling between the oss and cloud projects that we shouldn't have.